src/net.cpp

```diff
@@ -4237,7 +4237,7 @@ void CConnman::UnregisterEvents(CNode *pnode)
 {
 #ifdef USE_KQUEUE
     if (socketEventsMode == SOCKETEVENTS_KQUEUE) {
-        LOCK(pnode->cs_hSocket);
+        AssertLockHeld(pnode->cs_hSocket);
```

Comment:
Shouldn't we add EXCLUSIVE_LOCKS_REQURIED in header?

Reply:
We can't do this cause we lock `cs_hSocket` _per node_ and `UnregisterEvents()` is a member of `CConnman`, not `CNode`